felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1241,7 +1243,7 @@ func (m *bpfEndpointManager) calculateTCAttachPoint(policyDirection PolDirection
 		} else {
 			endpointType = tc.EpTypeTunnel
 		}
-	} else if ifaceName == "wireguard.cali" {
```

Comment:
I think we should preserve this, it is our hardcoded name and imo there is no need to expose it to the user.